docs/1-introduction/1-1-getting-started/1-1-3-hello-triangle.md

```diff
@@ -54,27 +71,43 @@ VSOutput Main(VSInput input)
 }
 ```
 
-First off, we define 2 types, `VSInput` and `VSOutput` which represent the vertex shader's input and output.
+First off, we define 2 types, `VSInput` and `VSOutput` which represent the 
+vertex shader's input and output.
+
+The input is 2, `float3` (vector of 3 float components), the first is the 
+"position" field, which are coordinates ranging from [-1.0, 1.0], if the values 
+are outside this range they are clipped, and we won't see them on screen.
 
-The input is 2, `float3` (vector of 3 float components), the first is the "position" field, which are coordinates ranging from [-1.0, 1.0], if the values are outside this range they are clipped, and we won't see them on screen.
+The second is the "color" field, which we also pass as the output of this stage 
+onto the pixel shader.
 
-The second is the "color" field, which we also pass as the output of this stage onto the pixel shader.
+Notice how all our fields have a colon and some identifier attached to them, 
+these are "semantics". Semantics that are preceded by `SV` are called 
+"system-value semantics" and their meaning and usage is defined by D3D11. 
+`SV_Position` for example means that the field `position` will be used by 
+D3D11 as the actual output of the vertex shader.
 
-Notice how all our fields have a colon and some identifier attached to them, these are "semantics".
-Semantics that are preceded by `SV` are called "system-value semantics" and their meaning and usage is defined by D3D11. `SV_Position` for example means that the field `position` will be used by D3D11 as the actual output of the vertex shader.
-Everything else are "user defined semantics" and their naming is up to us. These are used to pass data between shader stages.
+Everything else are "user defined semantics" and their naming is up to us. 
+These are used to pass data between shader stages.
 
-Then we have our `VSOutput`, which has our vertices in the first field `position` and our color in the second field `color`.
+Then we have our `VSOutput`, which has our vertices in the first field `position` 
```

Comment:
here we should mention SV_Position, NDC and potentially its effect on the zbuffer

Reply:
probably a bad time to explain all of this - the range of expected [-1.0, 1.0] is a fixed number - 
"if the `SV_Position` values are tried to be given a position outside the range of 
[-1.0, 1.0] they are clipped and we won't see them on screen. Due to this we need
to create our own math to transform any coordinates we have to a normalized [-1.0, 1.0]
This is going to be explored later in more depth."

due to this I think we should bring up that there is math we use later - since right now we are focusing on getting a triangle to draw, normalizing coordinates are  much better lumped up with the other explanations of  screen space, clip space, world space, etc. - explain it all once, not twice in 2 areas is my point